### PR TITLE
add ci file for flatpak

### PR DESCRIPTION
**Pull request by Murmele**
Add automatic build of the flatpak package

### CODE REVIEW

**Comment by kerizane** [comment]
Can you take a look at the failing build?

**Review by kerizane — CHANGES_REQUESTED** [review]
Fix the build.

**Comment by kerizane** [comment]
Well, you can validate the yaml using a tool like this: http://www.yamllint.com/

File: .github/workflows/build_flatpak.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Clone flatpak manifest from Flathub
+      uses: GuillaumeFalourd/clone-github-repo-action@v1
+      with:
+        owner: 'flathub'
+        repository: 'org.tuxemon.Tuxemon'
+
+    - name: Checkout development branch (TODO: remove this once a new version of tuxemon is released)
```
Comment (kerizane) [inline .github/workflows/build_flatpak.yml]: It doesn't like the colon after "TODO", change it to something else, e.g. "TODO - "

**Review by kerizane — CHANGES_REQUESTED** [review]

**Comment by Murmele** [comment]
On hold for now as long as the package is not published on flathub. Hopefully next days

**Review by kerizane — DISMISSED** [review]

**Comment by Murmele** [comment]
@kerizane it is building now. You can test the flatpak from the build artifacts: https://github.com/Tuxemon/Tuxemon/actions/runs/2780343304

**Review by kerizane — APPROVED** [review]

**Pull request merged**